myhpi/docker-compose.yml

```diff
@@ -16,7 +16,9 @@ services:
        volumes:
            - myhpi_postgres_data:/var/lib/postgresql/data/
        env_file:
-          - .env
+           - .env
+       environment:
+           - PGUSER=myhpi
```

Comment:
Various fixes but only 2 lines changed?

Reply:
yeah I was a bit confused by my git diff which showed more changes that somehow occured during testing locally. In the end, this ist just the postgres error message fix. I'm gonna squash the PR and use a better message there